.buildkite/shared/lib/json_tools.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
(I'm committing that other PR before this one, so upon commit this will all just disappear)

Reply:
I think a rebase would be cleaner 🙏